docs/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.lmchatgooglevertex.md

```diff
@@ -0,0 +1,36 @@
+---
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
+title: Google Vertex Chat Model
+description: Documentation for the Google Vertex Chat Model node in n8n, a workflow automation platform. Includes details of operations and configuration, and links to examples and credentials information.
+contentType: integration
+---
+
+# Google Vertex Chat Model
+
+Use the Google Vertex AI Chat Model node to use Google's Vertex AI chat models with conversational agents.
+
+On this page, you'll find the node parameters for the Google Vertex AI Chat Model node, and links to more resources.
+
+/// note | Credentials
+You can find authentication information for this node [here](/integrations/builtin/credentials/google/service-account/).
+///
+
+## Node parameters
+
+**Project ID**: the project ID from your Google Cloud account to use. n8n dynamically loads projects from the Google Cloud account, but you can also specify it manually.
+
+**Model Name**: the name of the model to use to generate the completion, for example `gemini-1.5-flash-001`, `gemini-1.5-pro-001`, etc. You can find the list of available models [here](https://cloud.google.com/vertex-ai/generative-ai/docs/learn/models).
+
+## Node options
+
+* **Maximum Number of Tokens**: change the maximum possible length of the completion.
+* **Sampling Temperature**: controls the randomness of the sampling process. A higher temperature creates more diverse sampling, but increases the risk of hallucinations.
+* **Top K**: the number of token choices the model uses to generate the next token.
+* **Top P**: use a lower value to ignore less probable options.
+* **Safety Settings**: Gemini supports adjustable safety settings. Refer to Google's [Gemini API safety settings](https://ai.google.dev/docs/safety_setting_gemini){:target=_blank .external-link} for information on the available filters and levels.
+
```

Comment:
## Templates and examples

<!-- see https://www.notion.so/n8n/Pull-in-templates-for-the-integrations-pages-37c716837b804d30a33b47475f6e3780 -->
[[ templatesWidget(title, 'google-vertex-chat-model') ]]